charts/stfc-cloud-openstack-cluster/bootstrap.sh

```diff
@@ -95,15 +95,15 @@ sudo microk8s status --wait-ready
 
 echo "Exporting the kubeconfig file..."
 mkdir -p ~/.kube/
-sudo microk8s.config  > ~/.kube/config
-sudo chown $USER ~/.kube/config
+echo microk8s.config | sudo tee ~/.kube/config > /dev/null
```

Comment:
I've just noticed this could blank an existing kubeconfig, if it's the users only copy and they run bootstrap....

We should backup their copy:

```
mkdir -p ~/.kube/
echo "Backing up existing kubeconfig if it exists..."
[ -f "~/.kube/config" ] || mv -v "~/.kube/config" ~/.kube/config.bak"
echo microk8s.config | sudo tee ~/.kube/config > /dev/null
```

If they run it twice then yeah the backup will be lost, but a simple safety net could go a long way

Reply:
oops @Dmitry-Popovichev i was not running the script but instead piping the name of the script into the file  - should be fixed now